e2e_test/source_inline/kafka/protobuf/basic.slt

```diff
@@ -50,9 +66,16 @@ select min(id), max(id), max((sc).file_name) from sr_pb_test;
 ----
 0 19 source/context_019.proto
 
+query TT
+select min(rw_key), max(rw_key) from sr_pb_upsert;
+----
+\x7b226964223a20307d \x7b226964223a20397d
```

Comment:
nit: `convert_from(rw_key, 'UTF-8')` to demonstrate how to see the key as readable json